portal-api/src/v1/validation/fields/issued-date.js

```diff
@@ -22,38 +19,42 @@ module.exports = (facility, errorList, deal) => {
     manualInclusionNoticeSubmissionDate: minSubmissionDateTimestamp,
   } = deal.details;
 
-  const dealSubmissionDate = formattedTimestamp(dealSubmissionDateTimestamp);
-  const minSubmissionDate = formattedTimestamp(minSubmissionDateTimestamp);
-  const issuedDate = formattedTimestamp(facility.issuedDate);
-  const today = moment();
+  const dealSubmissionDate = new Date(Number.parseInt(deal.details.submissionDate, 10));
+  const minSubmissionDate = new Date(Number.parseInt(deal.details.manualInclusionNoticeSubmissionDate, 10));
+  const issuedDate = new Date(Number.parseInt(facility.issuedDate, 10));
+  const today = new Date();
 
-  let dateOfSubmission = moment().utc().valueOf();
+  let dateOfSubmission = today.valueOf();
 
   if (dealSubmissionDateTimestamp || minSubmissionDateTimestamp) {
     if (submissionType === CONSTANTS.DEAL.SUBMISSION_TYPE.AIN) {
-      dateOfSubmission = dealSubmissionDate;
+      dateOfSubmission = new Date(dealSubmissionDate.valueOf());
     } else if (minSubmissionDateTimestamp) {
-      dateOfSubmission = minSubmissionDate;
+      dateOfSubmission = new Date(minSubmissionDate.valueOf);
     }
   }
 
-  if (dateHasAllValues(issuedDateDay, issuedDateMonth, issuedDateYear)) {
-    if (!moment(issuedDate).isSameOrAfter(dateOfSubmission, 'day')) {
-      const formattedDealSubmissionDate = moment(dateOfSubmission).format('Do MMMM YYYY');
+  const issuedDayHasValues = hasValue(issuedDateDay) && hasValue(issuedDateMonth) && hasValue(issuedDateYear)
+
+  // This is checking d,m,y and then doing logic with timestamp?
+  if (issuedDayHasValues) {
```

Comment:
This if statement checks that the `issuedDate-day`, `month` and `year` values exist but then using `issuedDate` timestamp. 

There should be validation elsewhere checking these are ok & adding `issuedDate` to the `facility` object but I'm not sure I trust this

Reply:
This is quite odd and something I've picked up on too. Could you raise a ticket for some kind of validation between the two (or just the removal of the day/month/year -- I'd leave the ticket open pending scoping)

Reply:
https://ukef-dtfs.atlassian.net/browse/DTFS2-7025